src/node_util.cc

```diff
@@ -10,6 +11,7 @@ using v8::Array;
 using v8::Boolean;
 using v8::Context;
 using v8::Function;
+using v8::FunctionTemplate;
```

Comment:
I believe the linter does not like the order here, this should be moved below `v8::FunctionCallbackInfo`?

Reply:
Done!